packages/analytics/src/utils/status.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+let AnalyticsEnabled = true;
+
+export const enableAnalytics = () => (AnalyticsEnabled = true);
+
+export const disableAnalytics = () => (AnalyticsEnabled = false);
```

Comment:
Nit: I think these would currently both return `true` (via returning the assignment)
```suggestion
export const enableAnalytics = () => { AnalyticsEnabled = true };

export const disableAnalytics = () => { AnalyticsEnabled = false };
```